### ISSUE TITLE
Screen reader is reading the table information when focus is on edit fields but visually not appearing as table present under “text”pane.

### ISSUE THREAD

**Issue by sagar17sep** [OfficeDev/office-ui-fabric-react#11030]
### Environment Information

- **Package version(s)**: 7.24.0
- **OS version:** 1903 (OS Build: 18326.295)
- **Browser:** Edge(Microsoft Edge 44.18362.267.0)
- **Screen Reader:** Narrator.

**User Experience:** 
If the screen reader is reading the table information while navigating on “Hex ,Red ,Green, Blue, Alpha” edit fields then screen reader dependent users might face difficulties to understand the data present  under “Text” pane.

**Repro steps:**
1. Open URL "https://make.test.powerapps.com" and login with valid credentials.
2. PowerApps Homepage will appear.
3. Navigate to template present at the bottom, then navigate to any already created portal app.
4. Navigate to context menu of portal app and select it. Select Edit (preview) from the context menu, portal will be opened on the screen.
5. Navigate to right pane 'Components' and select it.
6. Navigate to 'Text' under portal components on 'Components' pane and select it.
7. Changes will appear on right pane, navigate to right pane and try to edit Alignment, font color' of the text.
8. Navigate to “Hex ,Red ,Green, Blue, Alpha” edit fields” .
9. Observe the screen reader behavior.

**Actual behavior:**
Screen reader is reading the “Table” information while navigating in scan mode.

**Expected behavior:**
Screen reader shouldn’t read the table information while navigating on “Hex ,Red ,Green, Blue, Alpha” edit fields present under text pane. If table is defined for the design purpose then role="Presentation" should be defined.

**Please find the attachment**
[MAS1.3.1-text Pane table info..wmv.zip](https://github.com/OfficeDev/office-ui-fabric-react/files/3792621/MAS1.3.1-text.Pane.table.info.wmv.zip)

Please reach out to Portal FX team for more details.

**Comment by maxwellred** [comment]
Hi @sagar17sep , This issue involves a product, PowerApps, our team neither builds nor maintains. For issues related to our product you are welcome to submit an official issue [here](https://github.com/OfficeDev/office-ui-fabric-react/issues/new/choose) and supplying a codepen demo of the respective control found [here](https://developer.microsoft.com/en-us/fabric#/controls/web) accompanied with the necessary repro steps. This issue will be labelled as "Resolution: External" and we would appreciate your help by closing this issue yourself. Thank you very much for your interest and support in the Office UI Fabric React project!
